test/db/docker-compose.yml

```diff
@@ -3,7 +3,7 @@
 version: '3'
 services:
   rest:
-    image: postgrest/postgrest:v11.2.2
+    image: postgrest/postgrest:v12.2.0
     ports:
```

Comment:
That's the important part, everything else is tests updates due to changes in 01-dummy-data.sql